components/breadcrumb/index.css

```diff
@@ -18,35 +18,35 @@ governing permissions and limitations under the License.
   --spectrum-breadcrumbs-block-size-multiline: var(--spectrum-breadcrumbs-height-multiline);
 
   /* text regular */
-  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-small);
+  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-100);
   --spectrum-breadcrumbs-font-size: var(--spectrum-font-size-200);
   --spectrum-breadcrumbs-font-family: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
-  --spectrum-breadcrumbs-font-weight: var(--spectrum-font-weight-regular); /* placeholder for --spectrum-font-regular */
+  --spectrum-breadcrumbs-font-weight: var(--spectrum-regular-font-weight); /* placeholder for --spectrum-font-regular */
 
   /* text regular active item */
   --spectrum-breadcrumbs-font-size-current: var(--spectrum-font-size-200);
   --spectrum-breadcrumbs-font-family-current: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
-  --spectrum-breadcrumbs-font-weight-current: var(--spectrum-font-weight-bold); /* placeholder for --spectrum-font-bold */
+  --spectrum-breadcrumbs-font-weight-current: var(--spectrum-bold-font-weight); /* placeholder for --spectrum-font-bold */
 
   /* text compact */
   --spectrum-breadcrumbs-font-size-compact: var(--spectrum-font-size-100);
   --spectrum-breadcrumbs-font-family-compact: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
-  --spectrum-breadcrumbs-font-weight-compact: var(--spectrum-font-weight-regular); /* placeholder for --spectrum-font-regular */
+  --spectrum-breadcrumbs-font-weight-compact: var(--spectrum-regular-font-weight); /* placeholder for --spectrum-font-regular */
 
   /* text compact active item */
   --spectrum-breadcrumbs-font-size-compact-current: var(--spectrum-font-size-100);
   --spectrum-breadcrumbs-font-family-compact-current: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
```

Comment:
I think this placeholder note can be removed:

`/* placeholder for --spectrum-font-family-default */`